Buildings/Experimental/OpenBuildingControl/ASHRAE/G36/AtomicSequences/EconModulationMultiZone.mo

```diff
@@ -0,0 +1,194 @@
+block EconModulationMultiZone
+  "Outdoor and return air damper position modulation sequence for multiple zone VAV AHU"
+
+  parameter Real outDamConSigMax(min=0, max=1, unit="1") = retDamConSigMin
+  "Maximum control loop signal for the outdoor air damper";
+  parameter Real retDamConSigMin(min=0, max=1, unit="1") = 0.5
+  "Minimum control loop signal for the return air damper";
+  parameter Real yConSigMin=0 "Lower limit of controller output";
+  parameter Real yConSigMax=1 "Upper limit of controller output";
+  parameter Real controllerGain=1 "Gain of controller";
+
+  CDL.Interfaces.RealInput TSup(unit="K", quantity = "ThermodynamicTemperature")
+    "Measured supply air temperature" annotation (Placement(transformation(extent={{-160,-40},{-120,0}}),
+        iconTransformation(extent={{-120,50},{-100,70}})));
+  CDL.Interfaces.RealInput TCooSet(unit="K", quantity = "ThermodynamicTemperature")
+    "Supply air temperature cooling setpoint" annotation (Placement(transformation(extent={{-160,-10},{-120,30}}),
+        iconTransformation(extent={{-120,80},{-100,100}})));
+  CDL.Interfaces.RealInput uOutDamPosMin(min=0, max=1, unit="1")
+    "Minimum economizer damper position limit as returned by the EconDamperPositionLimitsMultiZone sequence"
+    annotation (Placement(transformation(extent={{-160,-120},{-120,-80}}),
+        iconTransformation(extent={{-120,-30},{-100,-10}})));    //fixme: add quantity for ALL damper positions?
+  CDL.Interfaces.RealInput uOutDamPosMax(min=0, max=1, unit="1")
+    "Maximum economizer damper position limit as returned by the EconEnableDisableMultiZone sequence. 
+    If the economizer is disabled, this value equals uOutDamPosMin"
+    annotation (Placement(transformation(extent={{-160,-90},{-120,-50}}),
+        iconTransformation(extent={{-120,0},{-100,20}})));
+  CDL.Interfaces.RealInput uRetDamPosMin(min=0, max=1, unit="1")
+    "Minimum return air damper position limit as returned by the EconEnableDisableMultiZone sequence"
+    annotation (Placement(transformation(extent={{-160,40},{-120,80}}),
+        iconTransformation(extent={{-120,-100},{-100,-80}})));
+  CDL.Interfaces.RealInput uRetDamPosMax(min=0, max=1, unit="1")
+    "Maximum return air damper position limit as returned by the EconEnableDisableMultiZone sequence."
+    annotation (Placement(transformation(extent={{-160,80},{-120,120}}),
+        iconTransformation(extent={{-120,-70},{-100,-50}})));
+
+  CDL.Interfaces.RealOutput yOutDamPos(min=0, max=1, unit="1") "Economizer damper position"
+    annotation (Placement(transformation(extent={{120,-30},{140,-10}}),
+        iconTransformation(extent={{100,-30},{120,-10}})));
+  CDL.Interfaces.RealOutput yRetDamPos(min=0, max=1, unit="1") "Return air damper position"
+    annotation (Placement(transformation(extent={{120,10},{140,30}}),
+        iconTransformation(extent={{100,10},{120,30}})));
+
+  CDL.Continuous.LimPID damPosController(
+    Td=0.1,
```

Comment:
The controller seems to be a bit inconsistent. I am using, per Brent's recommendation, a PI controller. If I set the controller type in ```LimPID``` to PI, both Td and Nd parameters become disabled (as expected). However, as I remove those params from my model, I get an error - only for Td. It turns out Nd has a default value of 10, whereas a default is not set for Td. On the fly I was not able to see why the controller still requires Td to get assigned a value, even though I'm using a PI controller, so ```with_d``` value is 0. For now I'm leaving Td in the code until I get your feedback.